_includes/sponsors.html

```diff
@@ -0,0 +1,16 @@
+<section class="tiers">
+    <div class="text-container">
+        {% for tier in site.data.sponsors %}
+        {% if tier[1].size > 0 %}
+        <div class="tier">
+            <div class="title">{{tier[0] | capitalize}} Sponsors</div>
+            <div class="sponsors">
+                {% for sponsor in tier[1] %}
+                {% include sponsor.html name=sponsor.name logo=sponsor.logo alt=sponsor.alt %}
```

Comment:
@johnmccabe Looks like the links are pointing at `href="#"` may I suggest to point them to each vendor and include a `?utm_campaign=openfaas` for example.

Reply:
I've fixed the URL, thanks for pointing that out.

Apparently `utm_*` needs a whole bunch of required parameters i.e. http://blog.rafflecopter.com/2014/04/utm-parameters-best-practices/